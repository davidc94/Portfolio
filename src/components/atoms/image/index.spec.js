import React from 'react';
import renderer from 'react-test-renderer';
import StyledImage from './index';

describe('StyledImage component', () => {
  it('render default StyledImage as expected', () => {
    const render = renderer.create(
      <StyledImage />
    );
    expect(render.toJSON()).toMatchSnapshot();
  });
});