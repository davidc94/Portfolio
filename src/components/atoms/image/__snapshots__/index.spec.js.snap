// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StyledImage component render default StyledImage as expected 1`] = `
<img
  className="sc-bdVaJa eLbGxn"
/>
`;
